app/scripts/models/reliers/sync.js

```diff
@@ -13,14 +13,24 @@
 define(function (require, exports, module) {
   'use strict';
 
+  var AuthErrors = require('lib/auth-errors');
   var _ = require('underscore');
   var Relier = require('models/reliers/relier');
   var ServiceNameTranslator = require('lib/service-name');
+  var Vat = require('vat');
+
+  /*eslint-disable camelcase*/
+  var QUERY_PARAMETER_SCHEMA = {
+    // context is not available when verifying.
+    context: Vat.string().min(1),
```

Comment:
No, once iframe support is deprecated in the OAuth flow, context will soon by a sync only feature. Keep it here.